src/auth/auth-get-uri-args.ts

```diff
@@ -0,0 +1,5 @@
+export interface BBAuthGetUriArgs {
```

Comment:
See the rename comment below regarding changing the file name and type name of this interface.